infrastructure/docker/docker-compose.builder.yml

```diff
@@ -19,7 +19,8 @@ services:
             - CONTINUOUS_INTEGRATION # Travis CI, Cirrus CI
             - BUILD_NUMBER # Jenkins, TeamCity
             - RUN_ID # TaskCluster, dsari
+            - HOME=/home/app
         volumes:
-            - "builder-data:/home/app"
             - "${COMPOSER_CACHE_DIR}:/home/app/.composer/cache"
             - "../..:/var/www:cached"
+            - "../../.home:/home/app:cached"
```

Comment:
Can you move this change to another PR? nothing to do here

Reply:
This change is needed for this PR, otherwise by default it would create a directory owned by root, i can make another PR but it would need to be merged before this one to avoid any problems